### PR TITLE
fix: respects DarkMode in CountryTableView

### PR DESCRIPTION
**Pull request by matheusmatos**
Hello guys, this simple PR fixes the issue https://github.com/firebase/FirebaseUI-iOS/issues/1013.

### CODE REVIEW

File: FirebasePhoneAuthUI/Sources/Resources/FUICountryTableViewController.xib [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="14835.7" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" colorMatched="YES">
```
Comment (matheusmatos) [inline FirebasePhoneAuthUI/Sources/Resources/FUICountryTableViewController.xib:2]: Changed automatically by xCode.

File: FirebasePhoneAuthUI/Sources/Resources/FUICountryTableViewController.xib [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="21701" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" colorMatched="YES">
     <device id="retina4_7" orientation="portrait" appearance="light"/>
     <dependencies>
-        <deployment identifier="iOS"/>
-        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="14790.5"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="21678"/>
+        <capability name="System colors in document resources" minToolsVersion="11.0"/>
```
Comment (matheusmatos) [inline FirebasePhoneAuthUI/Sources/Resources/FUICountryTableViewController.xib:6]: Changed automatically by xCode.

**Review by morganchen12 — APPROVED** [review]
Thanks!

**Comment by RaimundasSakalauskas** [comment]
Any estimation when this might get merged and shipped? Our users keep complaining about unusable country selector in dark mode.

**Comment by tehsven** [comment]
Hey @matheusmatos can you please hit the Merge button? This continues to be an issue and we're having to workaround it with some pretty hacky code.

**Comment by tehsven** [comment]
Or maybe @morganchen12 can merge this?

**Comment by vinhhoangbepos** [comment]
@morganchen12 @matheusmatos   Can you please merge this?

**Comment by RaimundasSakalauskas** [comment]
Since google team seems to be ignoring this entirely, here's the workaround that we have to make this work:
```
private var phoneAuthUIBugTask: Task<Void, Error>?
...
            if button.provider.providerID == PhoneAuthProviderID {
                phoneAuthUIBugTask = Task { @MainActor [weak self] in
                    while let self, self.presentedViewController != nil {
                        if let nvc = self.presentedViewController as? UINavigationController,
                           nvc.viewControllers.count >= 2, case let vc = nvc.viewControllers[1],
                           vc.view.subviews.count >= 1, let tableView = vc.view.subviews[0] as? UITableView {
                            tableView.backgroundColor = .systemBackground //fix dark mode
                            tableView.tableHeaderView = nil //remove search view, because it's broken
                        }
                        
                        //10x per second
                        try await Task.sleep(nanoseconds: 100_000_000)
                    }
                }
            }
...
```

I start the task when displaying the phone auth and cancel the task in `func authUI(_ authUI: FUIAuth, didSignInWith authDataResult: AuthDataResult?, error: Error?)`

**Review by morganchen12 — APPROVED** [review]

**Comment by morganchen12** [comment]
Hey everyone, apologies for the slow merge. This will be out in the next release.

**Pull request merged**